primitives/merkle-mountain-range/src/utils.rs

```diff
@@ -154,23 +211,5 @@ mod tests {
 				.collect::<Vec<_>>(),
 			sizes.clone()
 		);
-
-		// size cross-check
```

Comment:
I was thinking that since we test other cases that require for the size to be correctly cumputed (generating proofs), maybe we don't need this. But it's safer to have it. Added it back.